### PR TITLE
[WC-1349] virtual scroll issue - bump widgets version

### PR DESCRIPTION
**Pull request by gjulivan**
Bump version of affected widget in virtual scrolling issue

### CODE REVIEW

**Review by iobuhov — CHANGES_REQUESTED** [review]
What the reason for removing changelogs?

File: packages/modules/data-widgets/CHANGELOG.md [hunk 1]
```diff
@@ -6,10 +6,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
-### Changed
-
--   We replaced glyphicons with internal icons
-
```
Comment (iobuhov) [inline packages/modules/data-widgets/CHANGELOG.md:12]: Is this necessary?

Comment (gjulivan) [reply packages/modules/data-widgets/CHANGELOG.md:12]: it was removed because unecessary ?

**Review by iobuhov — APPROVED** [review]
LGTM

**Review by leonardomendix — APPROVED** [review]

**Review by leonardomendix — APPROVED** [review]

**Review by rahmanunver — APPROVED** [review]

**Pull request merged**